html/textroomtest.js

```diff
@@ -94,7 +96,7 @@ $(document).ready(function() {
 											});
 									}
 								},
-								ondataopen: function(data) {
+								ondataopen: function() {
```

Comment:
`ondataopen` is not a function without arguments:

	// Notify the open data channel
	pluginHandle.ondataopen(label, protocol);

The same comment applies to the other demos too, where I see the same change as been made. Maybe we were using a variable name that didn't make it clear, and ignoring the protocol part which was probably added later, but I think it's a mistake to not list them as attributes here, whether they're used or not.

Reply:
Removed since it was an unused param.
I'll add `label` and `protocol` params and the eslint directive to ignore the unused vars.